WordPressComStatsiOS/WordPressComStatsiOS.xcodeproj/xcshareddata/xcschemes/WordPressComStatsiOS.xcscheme

```diff
@@ -52,6 +52,36 @@
                ReferencedContainer = "container:WordPressComStatsiOS.xcodeproj">
             </BuildableReference>
          </TestableReference>
+         <TestableReference
```

Comment:
Hmmm interesting. Let me investigate why this delta occurred. At one point today all of the tests in XCode were disabled so I re-enabled them. I wonder if that caused this mutation.